.travis/run-ci.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -ex
+
+if [ -n "$STACK_YAML" ]; then
+  stack build --ghc-options -Werror --no-terminal
+  stack test --ghc-options -Werror --no-terminal
+elif [ -n "$CABAL_NEW_BUILD" ]; then
+  GHC_PATH=$(dirname $(stack exec which ghc))
+  export PATH=$GHC_PATH:$PATH
+  cabal update
+  cabal new-build
```

Comment:
I note that the stack build runs the tests. It would be worth adding `cabal new-test` here. It is probably best to new-configure as well. So I would change this to

```
cabal update
cabal new-configure --enable-tests
cabal new-build
cabal new-test
```

Reply:
I thought about this as well, but this PR is really just to make sure that people can use `generics-eot` as a dependency in `cabal new-build` projects. If users want to hack on `generics-eot` -- and therefore run the test suite -- I would expect them to use `stack`.